google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/data/v2/stub/SkipTrailersTest.java

```diff
@@ -252,14 +252,14 @@ public ServerServiceDefinition bindService() {
                   BigtableGrpc.getReadRowsMethod(),
                   ServerCalls.asyncServerStreamingCall(
                       (req, observer) -> rpcs.add(ServerRpc.create(req, observer))));
-      ImmutableList<MethodDescriptor<? extends GeneratedMessageV3, ? extends GeneratedMessageV3>>
+      ImmutableList<MethodDescriptor<? extends GeneratedMessage, ? extends GeneratedMessage>>
```

Comment:
This was the only necessary change to the handwritten layer (https://github.com/googleapis/java-bigtable/pull/2741/commits/0b26c75f0b738488c318c2836cbf6eaf567018cd)